lib/rubocop/cop/layout/first_array_element_line_break.rb

```diff
@@ -6,17 +6,49 @@ module Layout
       # Checks for a line break before the first element in a
       # multi-line array.
       #
-      # @example
+      # @example AllowMultilineFinalElement: false (default)
+      #
+      #     # bad
+      #     [ :a,
+      #       :b]
+      #
+      #     # bad
+      #     [ :a, {
+      #       :b => :c
+      #     }]
+      #
+      #     # good
+      #     [:a, :b]
+      #
+      #     # good
+      #     [
+      #       :a,
+      #       :b]
+      #
+      #     # good
+      #     [
+      #       :a, {
+      #       :b => :c
+      #     }]
```

Comment:
This looks awful. Is it really considered good by default now?

Reply:
Yeah this rule is only about having a line break before the first element in a multi line array, so it always considered this to be "good".